scripts/in_container/airflow_ci.cfg

```diff
@@ -1,59 +0,0 @@
-#
```

Comment:
@mobuchowski @amoghrajesh @hussein-awala @dstandish  -> we 've been discussing all the confusion about that one several times. Now this one is gone with single source of truth in `config_unit_tests.yml`. No more unittest.cfg file generated in AIRFLOW_HOME. defailts from config_unit_tests.yml  are loaded directly during tests.